src/index.html

```diff
@@ -5,9 +5,493 @@
+            <a href="#expertise" class="nav-mobile__link">Work</a>
+          </li>
+          <li class="nav-mobile__item">
+            <a href="#approach" class="nav-mobile__link">Services</a>
+          </li>
+          <li class="nav-mobile__item">
+            <a href="#vision" class="nav-mobile__link">Conctact us</a>
+          </li>
+        </ul>
+        <div class="nav-mobile__bottom">
+          <a href="tel:+1234555-55-55" class="nav-mobile__call">Hire Us</a>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section id="info" class="section section__info">
+        <div class="info">
+          <h2 class="info__heading section__heading_text">Who we are</h2>
+          <p class="info__text">
+            We embrace a strategic approach to creative ideas. We are<br>
+            interested in people and human relationships. This is the main<br>
+            thing you need to know about us. We believe in the power of<br>
+            bold ideas that can solve business challenges.
+          </p>
+        </div>
+      </section>
+      <section id="expertise" class="section section__expertise">
+        <h1 class="expertise__heading">Our exprertise</h1>
+        <div class="expertise__cards">
+          <div class="card">
+            <img
+              class="card__image"
+              src="./images/our-exprertise/image1.jpg"
+              alt=""
+            >
+            <h3 class="card__heading">Branding</h3>
+            <p class="card__text">
+              We create additional value for<br>
+              companies, products, services as well as<br>
+              verbal and visual ways to deliver it to<br>
+              the audience.
+            </p>
+          </div>
+          <div class="card">
+            <img
+              class="card__image"
+              src="./images/our-exprertise/image2.jpg"
+              alt=""
+            >
+            <h3 class="card__heading">Communication</h3>
+            <p class="card__text">
+              We strive to create communications that<br>
+              can increase media performance. We<br>
+              use everything — words, meanings,<br>
+              stories, art, movies.
+
+            </p>
+          </div>
+          <div class="card">
+            <img
+              class="card__image"
+              src="./images/our-exprertise/image3.jpg"
+              alt=""
+            >
+            <h3 class="card__heading">Strategy</h3>
+            <p class="card__text">
+              We create business growth strategies,<br>
+              from the moment of its birth to the<br>
+              achievement of the necessary business<br>
+              indicators.
+            </p>
+          </div>
+        </div>
+      </section>
+      <section id="process" class="section section__process">
+        <div class="process">
+          <div class="process__left">
+            <h3 class="process__heading section__heading">Process</h3>
```

Comment:
I guess, you couldn't use two different elements in BEM methodology (only 1 element and many blocks)
Use something like this `class="process__heading heading"`